css/css-position/sticky/position-sticky-stacking-context-002.html

```diff
@@ -25,14 +25,25 @@
 
   div#sticky
     {
-      background-color: green;
+      background: linear-gradient(to bottom, green 50%, red 50%);
       position: sticky;
       z-index: 2;
     }
+
+  div#overlap-bottom-half-of-sticky
+    {
+      background-color: green;
+      bottom: 50px;
+      height: 50px;
+      position: relative;
+      z-index: 3;
```

Comment:
If we're checking the stacking context of the sticky item, we need to not put z-index here. It needs to overlap even if there is no z-index, that's the point.

Reply:
I thought we could do both. A test with z-index: 2 value and a test without z-index. Anyway, I removed the 3 z-index specifications.